help/gmcommands.txt

```diff
@@ -0,0 +1,123 @@
+ <- @@index|<<Back to Index>>@@
```

Comment:
@o11c the @help command wither does not send all commands or manaplus truncates the message; I had to ask wushin for a copy of atcommand_local